cmd/zed/zed_conf.c

```diff
@@ -274,6 +276,9 @@ zed_conf_parse_opts(struct zed_conf *zcp, int argc, char **argv)
 		case 'd':
 			_zed_conf_parse_path(&zcp->zedlet_dir, optarg);
 			break;
+		case 'I':
+			zcp->do_idle = 1;
+			break;
```

Comment:
I forget this the first time around, but we're going to need to add this option to the `man/man8/zed.8` man page with a description.

Reply:
I couldn't find the "boo" emoji, so I updated the man page instead. :) Feed me sentences if they aren't clear enough.